packages/ui/src/overview/hooks/useDeadlines.ts

```diff
@@ -1,26 +1,42 @@
 import { useMemo } from 'react'
 
 import { useLocalStorage } from '@/common/hooks/useLocalStorage'
+import { useElectionStage } from '@/council/hooks/useElectionStage'
 import { asElection } from '@/council/types/Election'
+import { useMyMemberships } from '@/memberships/hooks/useMyMemberships'
+import { Member } from '@/memberships/types'
 import { useGetAllDeadLinesQuery } from '@/overview/queries'
+import { useWorkingGroups } from '@/working-groups/hooks/useWorkingGroups'
 import { asUpcomingWorkingGroupOpening } from '@/working-groups/types'
 
 interface UseDeadlinesParams {
   groupId?: string
+  member: Member
 }
 
-export type DeadlineNamespace = 'election' | 'proposals' | 'openings' | 'upcomingOpenings'
+export type DeadlineNamespace = 'elections' | 'proposals' | 'openings' | 'upcomingOpenings'
 
 const initializer: Record<DeadlineNamespace, string[]> = {
   proposals: [],
   openings: [],
   upcomingOpenings: [],
-  election: [],
+  elections: [],
 }
 
-export const useDeadlines = (params?: UseDeadlinesParams) => {
-  const options = params?.groupId || {}
-  const { loading, data } = useGetAllDeadLinesQuery(options)
+export const useDeadlines = (params: UseDeadlinesParams) => {
+  const { groups } = useWorkingGroups()
+  const { active: activeMembership } = useMyMemberships()
+  const groupIds = useMemo(
+    () => groups.flatMap((group) => (group.leadId === activeMembership?.id ? group.id : [])),
+    [activeMembership, groups.length]
+  )
+  const variables = {
+    proposalCreator: params.member.isCouncilMember ? undefined : { id_eq: params.member.id },
+    isLead: groupIds.length > 0,
+    group: groupIds.length > 0 ? { id_in: groupIds } : undefined,
+  }
+  const { stage: electionStage } = useElectionStage()
+  const { loading, data } = useGetAllDeadLinesQuery({ variables })
```

Comment:
```suggestion
  const { loading, data } = useGetAllDeadLinesQuery({ variables, skip: isLoadingWorkingGroup })
```

As a result the returned value for `isLoading` should now be:
```ts
  isLoading: isLoadingWorkingGroup || loading
```